tools/lsp/server_loop.rs

```diff
@@ -766,12 +660,11 @@ pub async fn remove_binding_command(
 /// Workaround for editor that do not support code action: using the goto definition on a comment
 /// that says "preview" will show the preview.
 fn maybe_goto_preview(
```

Comment:
(unrelated to this patch, but i think we can remove this workaround now.  Most editor support code actions by now.)